tests/test_main.py

```diff
@@ -61,12 +117,30 @@ def test_format_output_auto_expand():
         table_format='psql', dcmlfmt='d', floatfmt='g', max_width=100)
     table_results = format_output('Title', [('abc', 'def')],
                                   ['head1', 'head2'], 'test status', settings)
-    table = ['Title', '+---------+---------+\n| head1   | head2   |\n|---------+---------|\n| abc     | def     |\n+---------+---------+', 'test status']
+    table = [
+        'Title',
+        '+---------+---------+\n'
+        '| head1   | head2   |\n'
+        '|---------+---------|\n'
+        '| abc     | def     |\n'
+        '+---------+---------+',
+        'test status'
+    ]
     assert table_results == table
-    expanded_results = format_output('Title', [('abc', 'def')],
-                                     ['head1', 'head2'], 'test status', settings._replace(max_width=1))
+    expanded_results = format_output(
+        'Title',
+        [('abc', 'def')],
+        ['head1', 'head2'],
+        'test status',
+        settings._replace(max_width=1)
+    )
     expanded = [
-        'Title', u'-[ RECORD 1 ]-------------------------\nhead1 | abc\nhead2 | def\n', 'test status']
+        'Title',
+        u'-[ RECORD 1 ]-------------------------\n'
```

Comment:
It can sometimes lead to odd debugging situations when we mix unicode and byte strings. So, instead of sometimes using the `u` prefix and sometimes not, how about we just do `from __future__ import unicode_literals` at the top of the module so everything is unicode?